src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,8 +1,10 @@
 package racingcar
 
-data class Car(val position: Int = DEFAULT_POSITION) {
+data class Car(val name: String, val position: Int = DEFAULT_POSITION) {
 
-    fun move(isMove: Boolean): Car = if (isMove) Car(position + PLUS_POSITION) else this
+    fun move(isMove: Boolean): Car = if (isMove) Car(this.name, position + PLUS_POSITION) else this
+
+    fun isWinner(position: Int) = this.position == position
```

Comment:
```suggestion
    fun isWinner(position: Int) : Boolean = (this.position == position)
```

리턴 값에 대해 명시해 주고, 비교에 대해 한번 감싸줘서 표현하면, 코드에 대한 가독성이 더 좋아지지 않을까요? 🤔

Reply:
코틀린 스럽게 표현하려고 표현식에서 타입을 추론할 수 있게 해보려고 했는데 오히려 이러면 유지보수 측에서 가독성이 떨어질 것 같네요~